app/lib/services/sdk.ts

```diff
@@ -63,12 +64,13 @@ class Sdk {
 			MatchPathPattern<TPath>
 		>
 			? void
-			: Serialized<OperationParams<'POST', MatchPathPattern<TPath>>>
+			: Serialized<OperationParams<'POST', MatchPathPattern<TPath>>>,
+		apiVersion: string = 'v1'
 	): Promise<ResultFor<'POST', MatchPathPattern<TPath>>> {
 		return new Promise(async (resolve, reject) => {
 			const isMethodCall = endpoint?.startsWith('method.call/');
 			try {
-				const result = await this.current.post(endpoint, params);
+				const result = await this.current.post(endpoint, params, undefined, undefined, undefined, apiVersion);
```

Comment:
Commenting just so we're all aware, but this is only reasonable because there's a work [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5918) that would make it possible to call `sdk.post` specifying api version in a more elegant way.
`sdk.post('/v1/...`, `sdk.post('/apps/`...

Hopefully this is only temporary 😬